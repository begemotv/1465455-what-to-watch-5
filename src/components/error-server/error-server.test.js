import React from "react";
import renderer from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";

import ErrorServer from "./error-server";

test(`ErrorServer renders correctly`, () => {
  const tree = renderer
    .create(
        <MemoryRouter >
          <ErrorServer />
        </MemoryRouter >
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
