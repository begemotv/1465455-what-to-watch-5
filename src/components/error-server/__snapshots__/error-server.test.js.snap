// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorServer renders correctly 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title visually-hidden"
    >
      Loading error
    </h1>
  </header>
  <div
    className="sign-in__message"
  >
    <p>
      Sorry, there was an error on the server.
    </p>
    <p>
      Please try again.
    </p>
    <br />
    <a
      className="user-block__link"
      href="/"
      onClick={[Function]}
    >
      Go Home
    </a>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
